test/Sentry.DiagnosticSource.IntegrationTests/SqlListenerTests.verify.cs

```diff
@@ -155,7 +155,7 @@ public void ShouldIgnoreAllErrorAndExceptionIds()
         foreach (var field in eventIds)
         {
             var eventId = (EventId)field.GetValue(null)!;
-            var isEfExceptionMessage = SentryLogger.IsEfExceptionMessage(eventId);
+            var isEfExceptionMessage = Sentry.Extensions.Logging.SentryLogger.IsEfExceptionMessage(eventId);
```

Comment:
Hm, we've got two types called `SentryLogger` now? `Sentry.SentryLogger` and also `Sentry.Extensions.Logging.SentryLogger`? I can see that being a pain over time. I reckon we rename one of them. 

`Sentry.Extensions.Logging.SentryLogger` is `internal sealed` so no issues renaming that if we have to. `SentryMelLogger` maybe?

More painful would be things like [SentryLoggingOptions](https://github.com/getsentry/sentry-dotnet/blob/549bcd653c94b72e59e791a4503849188b2a48ca/src/Sentry.Extensions.Logging/SentryLoggingOptions.cs), since that's public. Definitely want to plan in advance to minimise ambiguity over naming once all of this is completed though.

Reply:
Are all of these public? `Sentry.Extensions.Logging.SentryLogger` 

https://github.com/getsentry/sentry-dotnet/blob/f765f991da77ecffd8813b772ddba55acc27d473/src/Sentry.Extensions.Logging/SentryLogger.cs#L6

This is internal so less of a concern

Reply:
We "leak" the internal `SentryLogger` name through the public `Sentry.Extensions.Logging.SentryLoggerProvider` and `Microsoft.Extensions.Logging.SentryLoggerFactoryExtensions` types. Well ... not really "leaking", but to some degree "implying".
So I would like to change the name of the new Logger.

Although I do like `SentryStructuredLogger`, ``M.E.L.ILogger`1`` also supports structured logging.

I go with `SentrySdkLogger` for now, definitely embracing the `[Experimental]`-Attribute, potentially changing it later when integrating it into `Sentry.Extensions.Logging` and using it from both `Sentry.AspNetCore` and `Sentry.Maui`, which I hope then makes the right name more obvious.

Reply:
I think `SentryStructuredLogger` is fine - it's our implementation of structured logging and it's not going to conflict with any other packages.

Reply:
renamed to `SentryStructuredLogger`
and moved to namespace `Sentry`